### DISCUSSION TITLE
How do I use supa_audit in supabase?

### DISCUSSION THREAD

**Discussion by carlos-alberto** [supabase/supabase#6253] [Questions]
Hey,

We're wanting to start using [supa_audit](https://github.com/supabase/supa_audit) in our Supabase database. Currently it doesn't seem to be available as an extension either in production or in the local CLI. Are there plans to make it available?

I found [this issue](https://github.com/supabase/supa_audit/issues/22) which suggests just manually running the script if on a managed platform, however there are no Supabase specific instructions. Is the expectation that we self manage supa_audit for now?

Thanks!

**Comment by MohammedFaragallah** [comment]
did you solve this?

I have the same problem i can't even build it locally because I'm using windows
